Makefile

```diff
@@ -106,7 +106,13 @@ $(ARTIFACTS):
 
 .PHONY: test
 test: ## Run tests
-	go test -v ./...
+	source ./scripts/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v ./...
```

Comment:
do we want to run test also include cover or just the test itself?

Reply:
Just the test, please compare with the main repo. I wouldn't want to do it differently here if we don't have to

Reply:
for the cover, there is another target.